src/main/java/org/prgrms/cream/domain/user/domain/User.java

```diff
@@ -99,4 +100,16 @@ public void updateUser(UserUpdateRequest userUpdateRequest) {
 			case SIZE -> this.size = userUpdateRequest.getValue();
 		}
 	}
+
+	public UserResponse toUserResponse() {
```

Comment:
```suggestion
	public UserResponse toResponse() {
```

User 엔티티라 이렇게만 해도 이해할 수 있을 것 같습니다!

Reply:
4fa30e7 : 아하.. User에서 응답객체로 바꾸는 것이니까 User라는 단어는 불필요하겠군요! 알려주셔서 감사합니다!!! 👍